docs/configuration.md

```diff
@@ -1752,6 +1752,14 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing sparkR shell in client modes for driver. Ignored in cluster modes. It is the same as environment variable <code>SPARKR_DRIVER_R</code>, but take precedence over it.
+    <code>spark.r.shell.command</code> is used for interactive mode of sparkR (sparkR shell) while <code>spark.r.driver.command</code> is used for the batch mode (running sparkR script).
```

Comment:
I mean if I change it to `interactive mode of sparkR shell`, this is a little weird to me as `sparkR shell` already means interactive mode.